documentation/developer-guide/modules/tooling-guide/pages/bamm-cli.adoc

```diff
@@ -3,81 +3,88 @@
+The available options and their meaning can also be seen in the help text of the individual subcommands.
 
-The following switches can be used in addition to the ones above to configure the generation:
-[width="100%", options="header", cols="40,60"]
+[width="100%",options="header",cols="20,50,30"]
 |===
-| Switch | Description
-| `--output-directory` or `-d` | Set the directory where files are written, defaults to the current directory. This
-  applies to all switches that generate files.
-| `--package-name` or `-pn` | Set the package name for generated Java files. This applies to `-java` and `-static-java`.
-| `--disable-jackson-annotations` or `-dja` | This option is to be used together with `-java` or
-  `-static-java` and will lead to generated Java code that does not contain
-  https://github.com/FasterXML/jackson[Jackson] Annotations.
-| `--execute-library-macros` or `-elm` | This option is used together with `-java` or `-static-java` and lead to generated Java code which includes the output from the custom macros provided via `-tlp` and `-tlfn`.
-| `--template-library-file` or `-tlf` | The path and name of the template file containing custom macros.
-It is used together with `-elm`.
-| `--use-semantic-api-version` or `-sv` | This option is used together with `-oapi-json` or `-oapi-yaml` in order to use
-  the complete semantic version of the Aspect Model as the version of the Aspect API.
-| `--aspect-api-base-url` or `-base-url` | This option is used together with `-oapi-json` or `-oapi-yaml` to provide the
-  base URL for the Aspect API OpenAPI specification.
-| `--include-aspect-query-api` or `-query-api` | This option is used together with `-oapi-json` or `-oapi-yaml` to include
-  the path for the Query Aspect API Endpoint in the OpenAPI specification.
-| `--aspect-exclude-paging` or `-no-paging` | This option is used to exclude the paging information in the OpenAPI specification. Paging is per default activated.
-| `--aspect-time-based-paging` or `-atbp` | This option is used to set the used paging strategy as time-based paging.
-| `--aspect-offset-based-paging` or `-aobp` | This option is used to set the used paging strategy as offset-based paging.
-| `--aspect-cursor-based-paging` or `-acbp` | This option is used to set the used paging strategy as cursor-based paging.
-| `--aspect-resource-path` or `-arp` | This option is used together with `-oapi-json` or `-oapi-yaml` to set the resource-path`
-  for the Aspect API endpoints.
-| `--aspect-parameter-file` or `-apf` | This option is used together with `-oapi-json` or `-oapi-yaml` to set a file including
-  the schema description for the resource. For JSON the description has to be in json, for YAML it has to be in YAML
+| Command | Description/Options | Examples
+| help                                  | Get overview of all commands                                   | bamm-cli help
```

Comment:
In this table, surround all example commands (in the last column) with \` \` so that they look `like this`

Reply:
Formatting added, but the commands in the table do not look like the commands with the same formatting outside of the table. Maybe the plugin I am using is showing it wrong. We need to generate the real thing and see if the formatting is right or not.